stages/org.osbuild.cloud-init

```diff
@@ -2,18 +2,9 @@
 """
 Configure cloud-init
 
-'configuration_files' option allows to create cloud-init `.cfg` configuration
-files under `/etc/cloud/cloud.cfg.d`. Its value is a dictionary which keys
-represent filenames of `.cfg` configuration files, which will be created.
-Value of each configuration file key is a dictionary representing the
-cloud-init configuration.
-
-Constrains:
-  - If 'configuration_files' option is specified, it must contain at least one
-    definition of a configuration file.
-  - Each configuration file definition must contain at least one configuration
```

Comment:
this constrain is still applicable in the new schema

Reply:
Sorry, but this one does not seem to be reolved...

Reply:
_puzzled look_ ... I am sure I did that. Let me fix it